### DISCUSSION TITLE
Ionic Components not rendering in GrapesJS canvas

### DISCUSSION THREAD

**Discussion by codevakure** [GrapesJS/grapesjs#5091] [Q&A]
Need help. I am trying to create ionic components drag and drop with GrapesJS.

So I added the styles and js to canvas.

**Tried this below**

    this.editor =grapesjs.init({
      container: '#editor',
      fromElement: 1,
      height: '100%',
      storageManager: { type: 0 },
      canvas: {
        styles: [
           "https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.esm.js",
        ],
        scripts: [
          { type: "module", src: "https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.esm.js" },  
        ]
      }
    });

**Added block**

    this.tcbeditor.BlockManager.add('ion-button', {
      id: 'ion-button',
      label: `Primary`,
      category: 'Buttons',
      content: `<ion-button fill="outline">Outline</ion-button>`,
      draggable: true,
      removable: true
    })
    
   But still ionic components are not rendering in the canvas.

**Comment by artf** [comment]
```diff
styles: [
- "https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.esm.js",
+ "https://cdn.jsdelivr.net/npm/@ionic/core/css/ionic.bundle.css",
],
```
